src/AirTable.php

```diff
@@ -160,14 +160,38 @@ public function createRows(array $rows, bool $typecast = true)
         foreach($rows as $row) {
             $data[] = ['fields' => $row];
         }
-        
+
         $this->chunkAndThrottle($data, function($rowsToCreate) use ($typecast) {
             $this->request('post', [
                 'records' => $rowsToCreate, 'typecast' => $typecast
             ]);
         });
     }
 
+
+    /**
+     * Update Rows.
+     *
+     * @param array $rows An array of data (see createRows for example)
+     * @param bool $typecast
+     */
+    public function updateRows(array $rows, bool $typecast = true)
+    {
+        $data = [];
+        foreach($rows as $row) {
```

Comment:
The data should be passed in in the correct format rather than having to loop to rearrange it. The `$rows` variable should be the format given in the AirtableHandler example on the ticket comment, e.g., `$rows` should be an array of ['id' => 'id or null', 'fields' => []].

I always feel manipulating arrays like this is somewhat asking for trouble, if we want an ID column we can't have it but that isn't enforced in the method signature, meaning you need knowledge of how the class works.

You'll also need to update createRows to make sure it uses the same format [id, fields] but only uses the fields as id is null

Reply:
This should be resolved as it's now handled before reaching this function.